File: WallBreakers/Arrays/TransposeMatric.java
class Solution {
    public int[][] transpose(int[][] A) {
      int rowNum = A.length, colNum = A[0].length;
      int[][] result = new int[colNum][rowNum];
      for (int row = 0; row < rowNum; row++) {
        for (int col = 0; col < colNum; col++) {
          result[col][row] = A[row][col];
        } 
      }
      return result;
    }
}

File: WallBreakers/Arrays/TransposeMatric.py
class Solution:
    def transpose(self, A: List[List[int]]) -> List[List[int]]:
      row, col = len(A), len(A[0])
      result = [[0] * row for _ in range(col)]
      for r in range(row):
        for c in range(col):
          result[c][r] = A[r][c]

      return result
